modules/emojifont/index.js

```diff
@@ -41,7 +41,7 @@ module.exports = class EmojifontModule extends BaseModule {
       this.bot.postMessage(data.channel, emojiFont.giveEmTheWorks());
     } else {
       try {
-        const resp = emojiFont.emojify(data.user_text);
+        const resp = data.user_text.split(' ').map(emojiFont.emojify).join(' ');
```

Comment:
this will leave us unable to match glyphs that have spaces in them :(
the reason spaces are a problem is how google sheet formats them

Reply:
there are glyphs with spaces in them!? oof

i'll be back with a new PR